<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>quersyStu</class>
 <widget class="QDialog" name="quersyStu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>519</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>481</width>
     <height>401</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>471</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QComboBox" name="cbb_method">
      <item>
       <property name="text">
        <string>请输入查找方式：</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/qT image/3.jpeg</normaloff>:/qT image/3.jpeg</iconset>
       </property>
      </item>
      <item>
       <property name="text">
        <string>按照姓名查找</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/qT image/11.jpeg</normaloff>:/qT image/11.jpeg</iconset>
       </property>
      </item>
      <item>
       <property name="text">
        <string>按照学号查找</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/qT image/12.jpeg</normaloff>:/qT image/12.jpeg</iconset>
       </property>
      </item>
      <item>
       <property name="text">
        <string>按照院系查找</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/qT image/4.jpeg</normaloff>:/qT image/4.jpeg</iconset>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="le_cnt"/>
    </item>
    <item>
     <widget class="QPushButton" name="btn_search">
      <property name="text">
       <string>查找</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
